antenna/app.py

```diff
@@ -5,128 +5,34 @@
-    global _LOGGING_SETUP
-    if _LOGGING_SETUP:
-        # NOTE(willkg): This makes it so that logging is only set up once per process.
-        return
-
-    host_id = app_config("host_id") or socket.gethostname()
-
-    class AddHostID(logging.Filter):
-        def filter(self, record):
-            record.host_id = host_id
-            return True
-
-    dc = {
-        "version": 1,
-        "disable_existing_loggers": True,
-        "filters": {"add_hostid": {"()": AddHostID}},
-        "formatters": {
-            "socorroapp": {
-                "format": "%(asctime)s %(levelname)s - %(name)s - %(message)s"
-            },
-            "mozlog": {
-                "()": "dockerflow.logging.JsonLogFormatter",
-                "logger_name": "antenna",
-            },
-        },
-        "handlers": {
-            "console": {
-                "level": "DEBUG",
-                "class": "logging.StreamHandler",
-                "formatter": "socorroapp",
-                "filters": ["add_hostid"],
-            },
-            "mozlog": {
-                "level": "DEBUG",
-                "class": "logging.StreamHandler",
-                "formatter": "mozlog",
-                "filters": ["add_hostid"],
-            },
-        },
-        "loggers": {
-            "antenna": {
-                "propagate": False,
-                "handlers": ["mozlog"],
-                "level": app_config("logging_level"),
-            }
-        },
-        "root": {"handlers": ["mozlog"], "level": "WARNING"},
-    }
-
-    if app_config("local_dev_env"):
-        # In a local development environment, we log to the console in a human-readable
-        # fashion and add a markus logger
-        dc["loggers"]["antenna"]["handlers"] = ["console"]
-        dc["loggers"]["markus"] = {
-            "propagate": False,
-            "handlers": ["console"],
-            "level": "INFO",
-        }
-        dc["root"]["handlers"] = ["console"]
-
-    logging.config.dictConfig(dc)
-    _LOGGING_SETUP = True
-
-
-def setup_metrics(metrics_classes, config, logger=None):
-    """Initialize and configures the metrics system."""
-    logger.info("Setting up metrics: %s", metrics_classes)
-
-    markus_configuration = []
-    for cls in metrics_classes:
-        backend = cls(config)
-        log_config(logger, backend)
-        markus_configuration.append(backend.to_markus())
-
-    markus.configure(markus_configuration)
-
-
-def log_config(logger, component):
-    """Log configuration for a given component."""
-    for namespace, key, val, opt in component.get_runtime_config():
-        if namespace:
-            namespaced_key = "%s_%s" % ("_".join(namespace), key)
-        else:
-            namespaced_key = key
-
-        namespaced_key = namespaced_key.upper()
-
-        if "secret" in opt.key.lower() and val:
-            msg = "%s=*****" % namespaced_key
-        else:
-            msg = "%s=%s" % (namespaced_key, val)
-        logger.info(msg)
```

Comment:
With Everett 2.0.0, this code is a little simpler and it's now in `liblogging.py`.